### DISCUSSION TITLE
Error in decompose(fnb_model) : time series has no or less than 2 periods

### DISCUSSION THREAD

**Discussion by Hendrychai** [teamalgoritma/community#804] [Q&A] [answered]
![image](https://github.com/teamalgoritma/community/assets/141151565/7935fc69-1066-420a-9f6c-17d80dd57274)

Frekuensi 1 karena datanya per jam, tapi error.

Bentuk data:
![image](https://github.com/teamalgoritma/community/assets/141151565/a9cfeaf6-15df-42f8-ab78-db9820e6b273)

**Comment by kevbow** [comment, marked as answer]
Dear @Hendrychai,

Decompose tidak dapat digunakan ketika menggunakan nilai `frequency = 1`. 
Jika Anda menggunakan data per jam maka frequency dapat disesuaikan dengan pola mana yang ingin ditinjau. Apabila ingin meninjau pola harian maka gunakan frequency 24, mingguan maka 24*7, dst.

Apabila jawaban ini membantu Anda silakan tekan `Mark as answer`. Terima kasih.

**Comment by kinantytasyao** [comment]
Halo @Hendrychai , karena tidak ada konfrimasi lebih lanjut dan untuk menjaga agar forum diskusi tetap berjalan kondusif, kami izin melakukan lock/close diskusi ini. Semoga dapat dipahami.
